### ISSUE TITLE
Arch Integration with Wiki

### ISSUE THREAD

**Issue by AmoleR** [vEnhance/otis-web#13]
Arch seems to be integrated with the wiki using the `problem` tag, but here are a few suggestions:
 - [x] Change syntax from `[problem PUID][/problem]` to [problem]PUID[/problem]`.
 - [x] Actually connect to problem database to access problem of said PUID.
 - [x] Allow a features flag (e.g. by default, problem will be hidden with a show/hide option. To directly show it, just do `[problem show]PUID[/problem]` or a similar syntax.

This issue isn't thought out in full, and I'll probably need @vEnhance input on how to connect to database (if possible). If anyone has anything else they want me to try to look at, let me know.

**Comment by vEnhance** [comment]
Blocked on #17

**Comment by vEnhance** [comment]
Marking this as done for now since the statement can now be integrated into the wiki page (and indeed, is by default).

Open a new issue for any further specific requests.
